sources/packages/backend/apps/api/src/auth/guards/institution/institution-student-data-access.guard.ts

```diff
@@ -17,13 +17,13 @@ export class InstitutionStudentDataAccessGuard implements CanActivate {
   ) {}
 
   async canActivate(context: ExecutionContext): Promise<boolean> {
-    const institutionStudentDataAccessParam =
-      this.reflector.getAllAndOverride<string>(
-        INSTITUTION_HAS_STUDENT_DATA_ACCESS_KEY,
-        [context.getHandler(), context.getClass()],
-      );
-
-    if (!institutionStudentDataAccessParam) {
+    const institutionStudentDataAccessParam = this.reflector.getAllAndOverride<
+      string[]
+    >(INSTITUTION_HAS_STUDENT_DATA_ACCESS_KEY, [
```

Comment:
Instead of a string array, I believe that we should create a type, similar to what we did below
https://github.com/bcgov/SIMS/blob/0fb865b5f458786721dab0ddaf78a397dd29560b/sources/packages/backend/apps/api/src/auth/decorators/institution/institution-location.decorator.ts